.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.log
 coverage/
 
 test/encrypted/nodejs-docs-samples.json
+dump.rdb
```

Comment:
Something created by redis-server when I run tests locally. It was bugging me, and I didn't want to accidentally commit it.